src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQueryBase.java

```diff
@@ -177,4 +177,27 @@ public void validateTheCmekKeyOfTargetBigQueryTableIfCmekIsEnabled(String cmek)
   public void waitForPageToRenderProperly() {
     SeleniumDriver.waitForPageToLoad();
   }
+
+  @Then("Enter BigQuery property {string} as macro argument {string}")
+  public void enterBigQueryPropertyAsMacroArgument(String pluginProperty, String macroArgument) {
+    enterPropertyAsMacroArgument(pluginProperty, macroArgument);
+  }
+
+  @Then("Enter BigQuery cmek property {string} as macro argument {string} if cmek is enabled")
+  public void enterBigQueryCmekPropertyAsMacroArgumentIfCmekIsEnabled(String pluginProperty, String macroArgument) {
+    String cmekBQ =  PluginPropertyUtils.pluginProp("cmekBQ");
+    if (cmekBQ != null) {
```

Comment:
similar comment here and for GCS as well

Reply:
Added log